database/database.go

```diff
@@ -149,15 +149,25 @@ func (db *Database) GetAllServiceVersions(id string) (services []servicemanager.
 }
 
 // ActivateService sets isActive to true for the service.
-func (db *Database) ActivateService(service servicemanager.ServiceInfo) (err error) {
+func (db *Database) ActivateService(serviceID string, aosVersion uint64) (err error) {
```

Comment:
please split changes into two commits.
1) change ActivateService API  and make appropriate changes in launched
2) [database] Add last used service timestamp